src/operator/numpy/np_elemwise_unary_op_basic.cc

```diff
@@ -92,6 +92,21 @@ bool NumpyUnaryLogicOpType(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+bool NumpyUnaryBoolOpType(const nnvm::NodeAttrs& attrs,
```

Comment:
I think you can directly replace `NumpyUnaryLogicOpType` above with this version